packages/flutter/lib/src/material/slider.dart

```diff
@@ -484,7 +484,7 @@ class Slider extends StatefulWidget {
   /// The cursor for a mouse pointer when it enters or is hovering over the
   /// widget.
   ///
-  /// If [mouseCursor] is a [WidgetStateProperty<MouseCursor>],
```

Comment:
I can imagine someone reading this and then being confused as to why their `WidgetStateProperty<MouseCursor>` object doesn't work as an argument for `mouseCursor`.

There are other similar files that include a description of `WidgetStateMouseCursor` as a footnote, but I think ideally the explanation should focus on this type.